scripts/dev-e2e.sh

```diff
@@ -0,0 +1,187 @@
+#!/bin/bash
+
+set -eo pipefail
+
+GREEN='\033[0;32m'
+RED='\033[0;31m'
+NC='\033[0m'
+
+STATUSFILE='armadactl-query.txt'
+AOREPO='https://github.com/armadaproject/armada-operator.git'
+AOHOME=../armada-operator
+ARMADA_CTL_VERSION='0.18.3'
+JOB_DETAILS=1
+
```

Comment:
Shall we delete the temp status file on exit?
```suggestion

trap 'rm -f -- "$STATUSFILE"' EXIT

```